### PR TITLE
Add more tokens to Home Page, Update FAQ + Litepaper

### PR DESCRIPTION
**Pull request by best-coder-NA**
Didn't include VSO because their logo isn't uploaded yet

Screen Shot 2021-06-05 at 4 21 45 PM

### CODE REVIEW

File: src/routes/faq.svelte [hunk 1]
```diff
@@ -38,7 +38,7 @@
     <FAQItem>
       <span slot="question">My transaction is pending on Metamask for some time now, what can I do?</span>
       <span slot="answer">
-        First check that your gas fee is set to exactly: 470 gwei. If it is, retry the transaction.
+        First check that your gas fee is set to exactly: 225 gwei. If it is, retry the transaction.
```
Comment (HariSeldon23) [inline src/routes/faq.svelte:41]: Good catch :)

File: src/lib/utils/constants.js [hunk 2]
```diff
@@ -12,7 +16,7 @@ export const TABLE_ADDRESSES = [
   "0xf39f9671906d8630812f9d9863bBEf5D523c84Ab",
   "0x39cf1BD5f15fb22eC3D9Ff86b0727aFc203427cc",
   "0x8cE2Dee54bB9921a2AE0A63dBb2DF8eD88B91dD9",
-  "0x99519AcB025a0e0d44c3875A4BbF03af65933627",
+  "0x99519AcB025a0e0d44c3875A4BbF03af65933627"
```
Comment (HariSeldon23) [inline src/lib/utils/constants.js:19]: Not sure if we have linting rules. So whether we end arrays with a comma or not? If we had a linting guide, we could just get it to enforce these rules

**Review by HariSeldon23 — COMMENTED** [review]
This looks good to me :) Nice work

File: src/lib/utils/constants.js (hunk 2 above)
Comment (bmino) [reply src/lib/utils/constants.js:19]: We do have prettier built into the project. Run this and it'll make a few changes:
@best-coder-NA 
```
npm run format
```

**Comment by bmino** [comment]
I'm looking at the token list PRs now too, so might be able to get that in as well

File: src/lib/utils/constants.js (hunk 2 above)
Comment (bmino) [reply src/lib/utils/constants.js:19]: I'd suggest leaving comments beside the TABLE_ADDRESSES too with the token name 😄

Comment (best-coder-NA) [reply src/lib/utils/constants.js]: Updated formatting and added comments!

**Pull request merged**